tests/integration_tests/integration_settings.py

```diff
@@ -15,6 +15,9 @@
 # Run tests marked as unstable. Expect failures and dragons.
 RUN_UNSTABLE = False
 
+# Instance type - is cloud specific. E.g., m6in.32xlarge on AWS/EC2
+INSTANCE_TYPE: Optional[str] = None
```

Comment:
I'm confused, what's up with line 35? I recall this being added before, but I never used it.